docs/development.md

```diff
@@ -0,0 +1,222 @@
+# Development
```

Comment:
ToC already contains the term 'dev server'. A term should be be used before it is defined.